gravitee-apim-e2e/api-test/use-case-test/src/management/publisher/endpoints/configure-lb-to-random-and-use-it.spec.ts

```diff
@@ -13,8 +13,101 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { describe } from '@jest/globals';
+import { APIsApi } from '@management-apis/APIsApi';
+import { forManagementAsApiUser } from '@client-conf/*';
+import { afterAll, beforeAll, describe, expect } from '@jest/globals';
+import { succeed } from '@lib/jest-utils';
+import { ApisFaker } from '@management-fakers/ApisFaker';
+import { ApiEntity } from '@management-models/ApiEntity';
+import { PlansFaker } from '@management-fakers/PlansFaker';
+import { LifecycleAction } from '@management-models/LifecycleAction';
+import { PlanStatus } from '@management-models/PlanStatus';
+import { fetchGatewaySuccess } from '@lib/gateway';
+import { teardownApisAndApplications } from '@lib/management';
+import { LoadBalancerTypeEnum } from '@management-models/LoadBalancer';
 
-describe('Configure LB to random and use it', () => {
-  test.skip('To complete', async () => {});
+const orgId = 'DEFAULT';
+const envId = 'DEFAULT';
+const apisResourceAsPublisher = new APIsApi(forManagementAsApiUser());
+
+describe('Configure LB to round robin and use it', () => {
```

Comment:
```suggestion
describe('Configure LB to random and use it', () => {
```

Reply:
Oops, ok, good catch, I'll fix and rebase it.